src/Compilers/CSharp/Portable/Binder/Semantics/Operators/OperatorKindExtensions.cs

```diff
@@ -152,7 +152,7 @@ public static BinaryOperatorKind WithType(this BinaryOperatorKind kind, SpecialT
                 case SpecialType.System_UInt64:
                     return kind | BinaryOperatorKind.ULong;
                 default:
-                    Debug.Assert(false, "Unexpected binary operator type.");
+                    Debug.Assert(false, _Unexpected);
```

Comment:
The previous error stated that this was a binary operator. This is now incorrect.